cluster-install/install.sh

```diff
@@ -2087,6 +2088,15 @@ step_collect_existing_trident_info() {
         logdebug "trident image: not found"
     fi
 
+    # Trident version
+    local -r trident_version="$(kubectl get tridentversions -n trident -o json | jq -r '.items.[0].trident_version' 2> /dev/null)"
+    if [ -z "$trident_version" ]; then
+        logwarn "Failed to resolve existing Trident version. ACP may not be supported without an upgrade!"
+    else
+        _EXISTING_TRIDENT_VERSION="$trident_version"
+        logdebug "trident version found: $trident_version"
```

Comment:
I think it's probably worth making this a `loginfo` TBH, reinforces to the user what their env setup currently is and all. 
```suggestion
        loginfo "* Trident version: $trident_version"
```